docs/kratos/organizations/organizations.mdx

```diff
@@ -348,6 +348,35 @@ curl -X PATCH --location "https://api.console.ory.sh/projects/$PROJECT_ID" \
 
 The SAML application callback URL to set at our SAML Identity Provider is: `https://api.console.ory.sh/saml/api/oauth/saml`
 
+#### Data Mapping
+
+You can configure the mapping of SAML attributes to Ory's identity schema using Jsonnet. All custom SAML attributes defined at the
+Identity Provider (IdP) will be available in the `raw_claims` object within `claims`. These attributes can then be mapped to the
+Ory identity schema.
+
+:::note
+
+Important: Ensure that the appropriate identity schema is created before mapping attributes. The email attribute is natively
+available in `claims.email`, everything else is nested under `claims.raw_claims`.
+
+:::
```

Comment:
```suggestion
:::warning

Ensure that the appropriate identity schema is created before mapping attributes. The email attribute is natively
available in `claims.email`, everything else is nested under `claims.raw_claims`.

:::
```

If you want to emphasize the importance here I would recommend to use "warning" or "danger" admonition - best use those if something will break prod if you get it wrong